src/components/Insights/TypeaheadSearch.js

```diff
@@ -86,14 +108,16 @@ export default class TypeaheadSearch extends React.Component {
         <span className="input-group-addon">
           <i className="fa fa-search"></i>
         </span>
-          <Autosuggest suggestions={suggestions}
-              inputProps={inputProps}
-              focusInputOnSuggestionClick={true}
-              onSuggestionSelected={(event, { suggestion })=>this.onSuggestionSelected(event, { suggestion })}
-              onSuggestionsFetchRequested={({value})=>this.onSuggestionsFetchRequested({value})}
-              onSuggestionsClearRequested={()=>this.onSuggestionsClearRequested()}
-              renderSuggestion={(element, query)=>this.renderSuggestion(element, query)}
-              getSuggestionValue={value => this.getSuggestionValue(value)} />
+          <Autosuggest
+            suggestions={suggestions}
+            inputProps={inputProps}
+            focusInputOnSuggestionClick={true}
+            onSuggestionSelected={this.onSuggestionSelected}
```

Comment:
Agree that this should be there but you forgot to bind `onSuggestionSelected` to the component in the constructor.

Reply:
This is not necessary given that it's now an arrow function.